README.md

```diff
@@ -1,2 +1,34 @@
-# wyrm
-LLVM-based realtime dsp language
+
+[comment]: # (Generated file, make edits to README.md.in)
+
+![wyrm logo](docs/static/images/wyrm_logo.png)
+
+# WYRM
+
+Wyrm is an experimental programming language for creative media production.
```

Comment:
I'm excited for Wyrm's broad goals but I think specifically mentioning audio and DSP would be a good idea here.